siriuspy/siriuspy/magnet/excdata.py

```diff
@@ -165,37 +166,39 @@ def __str__(self):
     # --- private methods ---
 
     @staticmethod
-    def _calc_interp(xt, yt, x):
-        interp = _np.interp(x, xt, yt,
-                            left=float('nan'), right=float('inf'))
+    def _calc_interp(xvals, xtab, ytab):
+        interp = _np.interp(
+            xvals, xtab, ytab, left=float('nan'), right=float('inf'))
         nan = _np.isnan(interp)
         inf = _np.isinf(interp)
-        v = _util.linear_extrapolation(x[nan],
-                                       xt[0], xt[1], yt[0], yt[1])
-        interp[nan] = v
-        v = _util.linear_extrapolation(x[inf],
-                                       xt[-1], xt[-2], yt[-1], yt[-2])
-        interp[inf] = v
+        vec = _util.linear_extrapolation(xvals[nan],
```

Comment:
doesn't pylint complain about this?
either way, I think it is better to put this in the standard style, like line 170 and 171 above.

Reply:
yes, pylint complains! the code was collapsed in the editor and I missed it. thanks.
fixed it, @fernandohds564.